.github/workflows/ubuntu.yml

```diff
@@ -18,10 +18,20 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
           architecture: 'x64'
+      - name: install dependcnies
+        run: sudo apt-get -y --no-install-recommends install libreoffice
```

Comment:
@w00lf is `libreoffice` required dependency for melanoma now?

I'm asking to understand should I add it to chocolatey / brew packages or not

Reply:
Yes, we want to test docx image extraction, without it it cannot be done.

Reply:
@camobap795 reverse_adoc is not a MN dependency, so libreoffice is not a MN dependency. No changes are needed in the packages.

Reply:
Libreoffice was always a reverse_adoc dependency.

Reply:
@ronaldtse it's only for `reverse_adoc` repo, not for the others, right?

Reply:
Correct.

Reply:
@CAMOBAP795  Does it mean that i dont need to change libreoffice install here? What should i do about this dependencies install next?

Reply:
> Does it mean that i dont need to change libreoffice install here?

Correct

> What should I do about this dependencies install next?

If this dependency used in this single repo, nothing need to be done

In case if it will be needed in another repos you will need to create directory in https://github.com/metanorma/metanorma-build-scripts/tree/master/ci-master/config/gh-actions (probably call it `libreoffice`) and put GH Actions configuration there + update https://github.com/metanorma/metanorma-build-scripts/blob/master/ci-master/config/ci.yml